dev/attest/cli.ts

```diff
@@ -59,8 +59,10 @@ if (testCmd === "node") {
         )
     }
     runTestsCmd += `node --loader ts-node/esm --test `
-} else {
+} else if (!process.versions.deno) {
```

Comment:
In general if there's not a reason to have the `if` condition be the positive version of the check, best to do it that way.